scripts/refresh-after-snapshot.py

```
@@ -0,0 +1,620 @@
+    print("  Keycloak sync complete")
+
+
+def create_secrets(config: RefreshConfig) -> None:
+    print("  Creating secrets...")
+    realm = json.loads(Path(config.realm_json).read_text())
+
+    fc_client = next(c for c in realm["clients"] if c.get("serviceAccountsEnabled"))
+    fc_id: str = fc_client["clientId"]
+    fc_secret: str = fc_client.get("secret", "")
+    if not fc_secret:
+        raise RuntimeError(f"No secret for client {fc_id} in realm.json")
```

Ruff (0.15.17)

[warning] Avoid specifying long messages outside the exception class

(TRY003)